cibuildwheel/util.py

```diff
@@ -2,12 +2,13 @@
 import urllib.request
 from fnmatch import fnmatch
 from time import sleep
-from typing import NamedTuple, List, Optional, Dict
+
+from typing import Dict, List, NamedTuple, Optional, Type, TypeVar
 
 from .environment import ParsedEnvironment
 
 
-def prepare_command(command, **kwargs):
+def prepare_command(command: str, **kwargs: str) -> str:
```

Comment:
str? `Dict`

Reply:
No: https://stackoverflow.com/questions/37031928/type-annotations-for-args-and-kwargs